python/ql/src/analysis/TypeHierarchyFailure.qhelp

```diff
@@ -4,11 +4,12 @@
 <qhelp>
 <overview>
 <p>In order to analyse uses of a class, all its attributes need to be known. Without the full inheritance hierarchy this is impossible.
-This is an informational query only. 
+This query finds classes where type inference fails and gives some indication as to why that failure occurred.
 </p>
 
 <p>
 This is an informational query only, this query depends on points-to and type inference.
+Unlike normal queries it does not indicate any problem with the program being analyzed.
```

Comment:
Possibly add: "It indicates a problem with the extraction of the code base."

Potentially, you might want to add a `<recommendation>` section suggesting that they look at customizing the Python extraction so that it's more complete?

Reply:
It doesn't necessarily indicate a problem with the code at all, merely that we can't analyze some part of the type hierarchy for the given reason.

Reply:
Okay - thanks for explaining.